webmessaging/messageerror.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<title>onmessageerror content attribute</title>
+<link rel="author" title="Domenic Denicola" href="mailto:d@domenic.me">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#handler-onmessageerror">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<body>
+
+<script>
+"use strict";
+window.messageErrorHappened = false;
+
+function cleanup() {
+  window.messageErrorHappened = false;
+  document.body.removeAttribute("onmessageerror");
+}
+
+test(() => {
+  assert_equals(document.body.onmessageerror, null, "body");
+  assert_equals(window.onmessageerror, null, "window");
+}, "The default value of onmessageerror is null");
+
+test(t => {
+  t.add_cleanup(cleanup);
+
+  document.body.setAttribute("onmessageerror", "window.messageErrorHappened = true;")
+  const compiledHandler = document.body.onmessageerror;
+
+  assert_equals(typeof compiledHandler, "function", "The onmessageerror property must be a function");
+  compiledHandler();
+  assert_true(window.messageErrorHappened, "Calling the handler must run the code");
+}, "The onmessageerror content attribute must be compiled into the onmessageerror property");
+
+test(t => {
+  t.add_cleanup(cleanup);
+
+  document.body.setAttribute("onmessageerror", "window.messageErrorHappened = true;")
+
+  document.body.dispatchEvent(new Event("messageerror"));
```

Comment:
I'm curious about this test -- if I try the same thing with `onmessage` and `message` instead, it doesn't fire on Chrome or Firefox. Is there something special about the messageerror event here? Or am I missing something?

(BTW, it works fine if the event is dispatched on window instead of document.body)